File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/preimage.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

sha3Hash = "0xaf953a2d01f55cfe080c0c94150a60105e8ac3d51153058a1f03dd239dd08586"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.preimage(sha3Hash)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/preimage.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const sha3Hash =
    "0xaf953a2d01f55cfe080c0c94150a60105e8ac3d51153058a1f03dd239dd08586";

  provider.debug
    .preimage(sha3Hash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
